plugins/protocolbuffers/kotlin/v21.10/buf.plugin.yaml

```diff
@@ -7,3 +7,17 @@ output_languages:
   - kotlin
 spdx_license_id: BSD-3-Clause
 license_url: https://github.com/protocolbuffers/protobuf/blob/v21.10/LICENSE
+registry:
+  maven:
+    compiler:
+      kotlin:
+        version: 1.8.10
+    deps:
+      - com.google.protobuf:protobuf-kotlin:3.21.10
+      - org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10
```

Comment:
Added jdk8 dependency to full runtime, and just normal stdlib to lite runtime. Does that sound reasonable @buildbreaker? I figure anyone using these dependencies can always pull in additional support for newer features.